src/request/sign-multisig-transaction/SignMultisigTransactionApi.js

```diff
@@ -141,10 +141,11 @@ class SignMultisigTransactionApi extends TopLevelApi {
             }
         } else if (object.secrets && typeof object.secrets === 'object'
             && 'encrypted' in object.secrets && 'keyParams' in object.secrets) {
-            // Not checking fixed length here, to stay flexible for future increases of the number of commitments
-            if (!Array.isArray(object.secrets.encrypted) || object.secrets.encrypted.length < 2) {
+            if (!Array.isArray(object.secrets.encrypted)) {
+                // No need to check object.secrets.encrypted.length here, as the number will later be checked against
```

Comment:
Although failing here would fail in the beginning of the request, instead of _after_ password entry.

Reply:
See https://github.com/nimiq/keyguard/pull/458#discussion_r2181074120